Marlin/src/gcode/queue.cpp

```diff
@@ -191,7 +191,6 @@ bool GCodeQueue::process_injected_command() {
   // Execute command if non-blank
   if (i) {
     parser.parse(cmd);
-    PORT_REDIRECT(SERIAL_PORT);
```

Comment:
This change should be fine, since injected commands occur outside the context of commands from serial port or media sources.